python/py-py7zr/Portfile

```diff
@@ -16,9 +16,11 @@ long_description    ${description}
 
 homepage            https://github.com/miurahr/py7zr
 
-checksums           rmd160  65cbfd2ce4b44ed8cae54c17ae1e64a5c02d6db7 \
-                    sha256  6986e90ca4ca1f4999d57987a54d891673c9f8befa02c9c6cd8201893a17442c \
-                    size    4255058
+checksums           rmd160  e71df6f0fd74398f9547ad28a0a1fad93fa9497b \
+                    sha256  81ebb10619739764c2d80ef222e5845373ce84523183371ae9cd88e0df908929 \
+                    size    4260349
+
+python.pep517       yes
```

Comment:
when adding this you can remove remove to explicit build dependency on `setuptools` below as this added automatically by the `python` PG (you will need the `setuptools_scm` dependency though)

Reply:
@reneeotten it's done

Reply:
ehm, I still see `port:py${python.version}-setuptools` listed below under `depdends_build-append`

Reply:
@reneeotten indeed, forgotten push. Sorry.